test/test.js

```diff
@@ -1164,12 +1067,10 @@ describe("S3rver Tests", function() {
     }
     async.eachSeries(
       testObjects,
-      function(testObject, callback) {
-        s3Client.putObject(testObject, function(err, object) {
+      (testObject, callback) => {
+        s3Client.putObject(testObject, (err, object) => {
+          if (err) return done(err);
```

Comment:
Probably still want to call `callback` instead of `done` here so that the `eachSeries` doesn't continue after the test fails.

Reply:
Wow, good eye!